src/code.lisp

```diff
@@ -58,17 +109,25 @@
         :query (read-query object)
         :header (vellum.header:read-header object)))
 
+(defun postgres-prepared-query? (object)
+  (eq (type-of object) 'postgres-prepared-query))
+
 
 (defmethod vellum:copy-from ((format (eql ':postmodern))
                              input
```

Comment:
I think that it would be much better to stick with input being either string or list. You can use (parameters '() parameters-bound-p) in the lambda list to know if user passed parameters or we are using the default value.